### PR TITLE
Footer

### LINKED ISSUE #28: make a wave animation based footer
**Is your feature request related to a problem? Please describe.**
make a component inside ```src/components``` of folder ```hotelontouch``` , name the file ```Footer.js``` and the function name should be ```Footer``` and it should have the link to the the Github Repository

Make the footer which looks nice, new and creative ideas are most welcome

### PR DESCRIPTION
**Pull request by batman005**
Fixes #28 
added a wave Footer with material ui icons 
Screenshots of the changes (If any) -
wave footer

Note: Please check Allow edits from maintainers. if you would like us to assist in the PR.

#### ✅️ By submitting this PR, I have verified the following

- [x] Worked on forked repo and starred the main repo
- [x] Checked to see if a similar PR has already been opened 🤔️
- [x] Reviewed the contributing guidelines 🔍️
- [x] Sample preview link added (add the link(s) for all the pages changed/updated from the checks tab after checks complete)
- [x] Tried Squashing the commits into one

### CODE REVIEW

File: hotelontouch/src/App.js [hunk 1]
```diff
@@ -1,9 +1,8 @@
 import React from 'react';
 import { MemoryRouter, Switch, Route } from 'react-router-dom';
-
 import { HomePage } from './pages';
 import { NavbarComponent } from './components';
-
+import Footer from './components/Footer.js';
```
Comment (ayan-biswas0412) [inline hotelontouch/src/App.js:5]: export the component in ```components/index.js``` as ```export { dafault as Footer } from './Footer';``` and then import here as ```import Footer from './components';```

**Review by ayan-biswas0412 — CHANGES_REQUESTED** [review]

**Review by ayan-biswas0412 — CHANGES_REQUESTED** [review]
And also see your CI checks are failing fix them

**Comment by batman005** [comment]
okay I will try to fix those issue ,btw thank you for reviewing it

**Comment by aniketsingh98571** [comment]
@batman005 , remove all the whitespaces which are there in your files, if you have any link with `href="#"` replace it with `href="/"`

**Comment by ayan-biswas0412** [comment]
> @batman005 , remove all the whitespaces which are there in your files, if you have any link with `href="#"` replace it with `href="/"`

Good @aniketsingh98571 to review works in this repository 🥰

**Comment by batman005** [comment]
@aniketsingh98571  thankyouu for the tip

**Comment by aniketsingh98571** [comment]
@batman005  please refer to #45 for CI checks

**Comment by batman005** [comment]
@ayan-biswas0412    I have made all the changes now the footer will work in all the devices  ipad,mobile,laptop,desktop at 100% zoom ,please have a look

**Review by ayan-biswas0412 — CHANGES_REQUESTED** [review]
your footer height is so large please reduce it

**Comment by ayan-biswas0412** [comment]
> @ayan-biswas0412 I have made all the changes now the footer will work in all the devices ipad,mobile,laptop,desktop at 100% zoom ,please have a look.
> ( I think we can also add payment types to the website and link into the footer )

yeah it is a good idea and please open an issue regarding this with proper implementation

**Comment by batman005** [comment]
> your footer height is so large please reduce it

reduced the size of footer ,have look..

**Comment by batman005** [comment]
> > @ayan-biswas0412 I have made all the changes now the footer will work in all the devices ipad,mobile,laptop,desktop at 100% zoom ,please have a look.
> > ( I think we can also add payment types to the website and link into the footer )
> 
> yeah it is a good idea and please open an issue regarding this with proper implementation

okay great, but it will take time because I am in the middle of my semester exams...

**Review by ayan-biswas0412 — CHANGES_REQUESTED** [review]
again the white space issue is not resolved, please resolve that otherwise I can't merge this

**Comment by batman005** [comment]
mobile view (iphone X) 100% zoom
iphone view
ipad view 100% zoom
ipad view
ipad pro view 100% zoom
ipad pro view
desktop view at full scroll
desktop view

@ayan-biswas0412  sir I used an  SVG so it is a static  wave because when we are using  react-wavify with style-component we aren't able to stick the footer at the bottom of  web app with the animation (example devices :- desktop(24 inch monitor), ipad and i padpro )
I think we should use an alternative for animation like CSS...

**Comment by ayan-biswas0412** [comment]
> mobile view (iphone X) 100% zoom
> iphone view
> ipad view 100% zoom
> ipad view
> ipad pro view 100% zoom
> ipad pro view
> desktop view at full scroll
> desktop view
> 
> @ayan-biswas0412 sir I used an SVG so it is a static wave because when we are using react-wavify with style-component we aren't able to stick the footer at the bottom of web app with the animation (example devices :- desktop(24 inch monitor), ipad and i padpro )
> I think we should use an alternative for animation like CSS...

if you want to use another alternative you may use that

**Comment by batman005** [comment]
wave in desktop
wave animation in desktop
iphone static
normal static footer in mobile view

@ayan-biswas0412  sir can we go for this type of footer option

**Comment by ayan-biswas0412** [comment]
> wave in desktop
> wave animation in desktop
> iphone static
> normal static footer in mobile view
> 
> @ayan-biswas0412 sir can we go for this type of footer option like static Footer in mobile view and wave animation footer in desktop ipad etc. i means in landscape mode.

no dear it is highly discouraged to change style according to the device

**Comment by batman005** [comment]
> > wave in desktop
> > wave animation in desktop
> > iphone static
> > normal static footer in mobile view
> > @ayan-biswas0412 sir can we go for this type of footer option like static Footer in mobile view and wave animation footer in desktop ipad etc. i means in landscape mode.
> 
> no dear it is highly discouraged to change style according to the device

@ayan-biswas0412  okay sir,I have made some changes please have a look into it.....

**Comment by batman005** [comment]
@ayan-biswas0412  okay sir, I  have made some changes  on the footer please  look into it..
(sir i have a doubt also do we get mentors in  gssoc 2021? )

**Comment by ayan-biswas0412** [comment]
> @ayan-biswas0412 okay sir, I have made some changes on the footer please look into it..
> (sir i have a doubt also do we get mentors in gssoc 2021? )

mobile view is still very scattered and not well resized

**Comment by ayan-biswas0412** [comment]
> @ayan-biswas0412 okay sir, I have made some changes on the footer please look into it..
> (sir i have a doubt also do we get mentors in gssoc 2021? )

regarding your doubt I think I am not the right person to tell that you can clarify your doubt from the GSSoc officials only

**Pull request closed without merging**